Assets/Shopify/UIToolkit/Test/Integration/TestRemoteImageLoaderIntegration.cs

```diff
@@ -0,0 +1,33 @@
+namespace Shopify.UIToolkit.Test.Integration {
+	using UnityEngine;
+    using UnityEngine.TestTools;
+    using System.Collections;
+    using Shopify.Unity.SDK;
+
+	[IntegrationTest.DynamicTest("UIToolkitIntegrationTests")]
+    public class TestRemoteImageLoaderIntegration {
+
+        [UnityTest] 
+        public IEnumerator TestDownloadImage() {
+            var loader = GlobalGameObject.AddComponent<RemoteImageLoader>();
+            UnityImageLoader networkLoader = new UnityImageLoader();
+            loader.SetImageLoader(networkLoader);
+
+            string url = "https://cdn.shopify.com/s/files/1/2094/7261/products/product-image-305453751.jpg?v=1497377691";
+
+            Texture2D downloadedTexture = null;
+            string returnedError = null;
+            loader.LoadImageURL(url, (texture, error) => {
+                downloadedTexture = texture;
+                returnedError = error;
+            });
+
+            while (downloadedTexture == null) {
```

Comment:
if the network is borked for whatever reason this test will run forever... maybe just make a temp bool called `requestFinished` and wait on that instead?